### ISSUE TITLE
"Error: Failed to open controlling terminal with error: input/output error" on kitty terminal

### ISSUE THREAD

**Issue by sr-prog** [NikitaIvanovV/ctpv#53]
![image](https://user-images.githubusercontent.com/117561994/236635689-cdae59f0-4724-4213-85bc-7f6cbce99f3a.png)

This error appears when I launch lf while using ctpv, I'm currently using kitty 0.28.1 and I think this error comes on versions that are higher than kitty 0.26.5 but I'm not 100% sure about it. Also I'm using icat for previews not ueberzug. Also image preview works completely fine but this error message coming up every time I launch lf is annoying and I'm not sure what's causing it to appear. Also I have tested and this error is there when I load kitty without any custom config.

**Comment by tionis** [comment]
I'm seeing a similar error in Kitty when using ctpv in a fzf preview command (with a wrapper that starts and stops the daemon in the background and sets the sizes)

**Comment by sr-prog** [comment]
So I figured out what was wrong, it's because of this command:
![image](https://github.com/NikitaIvanovV/ctpv/assets/117561994/4653425b-c57d-4b26-8429-159a5f8956d6)

I saw the readme and it looks like maybe this command is deprecated or something in favor of a ctpvquit command so I just replaced this in my lfrc

![image](https://github.com/NikitaIvanovV/ctpv/assets/117561994/98c8bc2c-521a-4d69-b26c-3cbc9b7978be)

with this:

![image](https://github.com/NikitaIvanovV/ctpv/assets/117561994/881377cb-65dd-4494-8f6b-891f7c88289c)

TLDR; replace ctpv -e with ctpvquit I guess

If this fix doesn't work for anyone else who sees this I can re-open the issue on your request.

**Comment by tionis** [comment]
I'm still experiencing this error when used in fzf previews, but that may as well be a problem with fzf itself

**Comment by NikitaIvanovV** [comment]
`ctpvquit` is a wrapper around `ctpv -e` which just checks an lf process is running and if it's not, it kills the background process. I added it because `cmd on-quit` is a pretty new addition to lf and a lot of people don't have it. Maybe you also have an old version of lf, which caused the issue?

**Comment by tionis** [comment]
The issue only happens in kitty.
I just used the [fzfp script in the stpv repo](https://github.com/Naheel-Azawy/stpv/blob/master/fzfp) and replaced the calls to stpv with appropiate calls to ctpv
